### ISSUE TITLE
Commit messages are truncated when they contain a `'` character

### ISSUE THREAD

**Issue by gekoke** [Artawower/blamer.el#54]
![image](https://github.com/Artawower/blamer.el/assets/63151928/1b57c955-82f5-45af-b244-4708cbd8cb6c)
![image](https://github.com/Artawower/blamer.el/assets/63151928/1ecb4d50-c973-49e7-ada1-28cf932dd43b)

My configuration:
```elisp
(use-package blamer
  :custom
  (blamer-idle-time 0.3)
  (blamer-min-offset 8)
  (blamer-max-commit-message-length 120)
  :custom-face
  (blamer-face ((t :background nil
                   :weight semi-light
                   :italic t)))
  :config
  (global-blamer-mode 1))
```

**Comment by Artawower** [comment]
Should have been fixed in 0.7.4

**Comment by gekoke** [comment]
> Should have been fixed in 0.7.4

Thanks! I'll check test it soon. Great package!

**Comment by gekoke** [comment]
works :)
